apps/api/v2/src/modules/organizations/organizations.module.ts

```diff
@@ -143,6 +149,7 @@ import { forwardRef, Module } from "@nestjs/common";
     OrganizationsTeamsSchedulesController,
     OrganizationsUsersOOOController,
     OrganizationsConferencingController,
+    OrganizationsStripeController,
```

Comment:
I don't know how and why but we have ended up with a huge `organizations.module.ts`. Can you create a new `organizations-stripe.module.ts` where you setup an isolated module that then is imported into the `organizations.module.ts` just like with `organizations-organizations.module.ts`.